src/pages/en/guides/cms/buttercms.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ButterCMS & Astro
+description: Add content to your Astro project using ButterCMS
+layout: ~/layouts/CMSLayout.astro
+setup: |
+    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+stub: false
+---
+
+ButterCMS is a headless CMS and blog engine that allows you to publish structured content to use in your project.
+
+## Integrating with Astro
+
+In this section, we'll use the ButterCMS SDK to bring your ButterCMS data into your Astro project.
```

Comment:
```suggestion
In this section, we'll use the [ButterCMS SDK](https://www.npmjs.com/package/buttercms) to bring your data into your Astro project.
```
ButterCMS twice sounded a little clunky? Also included a link to the SDK. maybe?